fixieai/cli/auth/oauth_flow.py

```diff
@@ -0,0 +1,65 @@
+import secrets
+import socket
+import webbrowser
+
+import click
+from oauth2_client import credentials_manager
+
+from fixieai import constants
+
+# Fixie CLI Client ID.
+CLIENT_ID = "II4FM6ToxVwSKB6DW1r114AKAuSnuZEgYehEBB-5WQA"
+# Scopes to request access for.
+SCOPES = ["fixie-api-key"]
+# The authorization URL that users click on.
+AUTHORIZE_SERVICE = f"{constants.FIXIE_API_URL}/authorize"
+# The token exchange service that we use internally.
+TOKEN_SERVICE = f"{constants.FIXIE_API_URL}/oauth/token"
```

Comment:
I think I would name this URL differently to avoid confusion with our /oauth redirect uri.